docs/contributing/testing.md

```diff
@@ -0,0 +1,152 @@
+---
+id: testing
+title: Contributing
+sidebar_label: Testing
+---
+
+# Testing
+
+When building Web sites and apps on FAST-DNA it's recommended to perform exhaustive testing to instill the confidence that you've met your customers' expectations. The checklist below was compiled over the last few years and is a good starting point for manual testing Web applications.
```

Comment:
Again, why are we speaking about general integrations here? Why is this file not speaking to how we test FAST-DNA itself?

Reply:
it does further below, and because we haven't flushed out how exactly we're going to test all of our packages.

Reply:
Will edit as we go forward.